.github/workflows/main.yml

```diff
@@ -0,0 +1,64 @@
+name: Tests
```

Comment:
Is this CI specifically for tests?

Reply:
Yes. We will have a separate CI yaml that does other things such as push to dockuerhub and push to pypi.